### ISSUE TITLE
Print expanded SQL statement

### ISSUE THREAD

**Issue by Zwelf** [ddnet/ddnet#2673]
The MySQL C++ doesn't provide a way to print a prepared statment AFAIK, but SQLite provides a function to [expand a prepared statement](https://www.sqlite.org/c3ref/expanded_sql.html). Therefore we could just always create a SQLite statement to print it to the console.

**Comment by edg-l** [comment]
The point of printing the statement is for debugging purposes right? Couldn't it be the case that mysql expands the statement in another way than sqlite?

**Comment by Learath2** [comment]
It's a bad idea to expand this ourselves except for maybe ONLY debugging
purposes. Especially when we are working with unicode names. PHP had a
handful of serious bugs from their simulated prepared statements

**Comment by Zwelf** [comment]
Yes expanding the statement would just be to print it and as a last resort to insert it to the database. MySQL doesn't expand the query before sending it to the MySQL server. It just sends the variables separately.
I wouldn't categorize this as expanding the query ourselves, since SQLite provides a the function to expand the SQL query. Therefore printing the SQL statement in the SQLite backend would be easy, but hard for the MySQL statements (would need to simultaneously build the statement in SQLite, might not be desirable)
